css/index.css

```diff
@@ -1,3 +1,520 @@
+        display: flex;
+        flex-direction: column;
+    
+    }
+
+    header nav {
+        height: 60px;
+        width: 800px;
+        display: flex;
+        position: relative;
+        line-height: 25px;
+        flex-direction: row;
+        
+    }
+    
+    header nav a {
+        margin: 95px 55px 0 28px;
+        text-decoration: none;
+        justify-content: flex-start;
+        color: rgb(159, 159, 180);
+        line-height: 36px;
+        font-size: 16px;
+        font-weight: bold;
+    }
+
+    
+    header .logo {
+        margin-bottom: 65px;
+        height: 36px;
+        width: auto;
+        align-self: center;
+    }
+    
+    .main-content .snippy {
+        display: none;
+    }
+    
+    .main-content .stack-text {
+        position: absolute;
+        margin-top: 130px;
+        margin-left: 160px;
+        height: 310px;
+        text-align: center;
+        line-height: 73px;
+        font-size: 73px;
+    }
+
+    .main-content .start {
+        position: absolute;
+        border: 1px solid gray;
+        left: 315px;
+        bottom: 67px;
+        padding: 8px 40px 3px 43px;
+        height: 33px;
+        font-size: 17px;
+        text-align: center;
+        font-weight: bold;
+	}
+	
+	.toppy {
+		border-bottom: 2px solid black;
+		margin: 7px 21px 0px 21px;
+	}
+
+	h2 {
+		font-size: 16px;
+	}
+	
+	.info-content .feature-about {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+	
+	.info-content .feature {
+		padding-top: 37px;
+		margin-right: 10px;
+		margin-left: 30px;
+		width: 350px;
+		line-height: 18px;
+		font-size: 16px;
+	}
+	
+	.info-content .about {
+		padding-top: 37px;
+		width: 370px;
+		line-height: 18px;
+	}
+	
+	.middle-img {
+		margin: 30px 21px 30px 11px;
+		height: 156px;
+		width: 760px;
+	}
+	
+	.info-content .ser-pro-vis {
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+	
+	.info-content .ser-pro-vis .services {
+		margin-right: 0px;
+		margin-left: 30px;
+		padding-right: 20px;
+		height: 200px;
+		width: 235px;
+		padding-bottom: 0px;
+		line-height: 18px;
+	}
+	
+	.info-content .product {
+		margin: 0 0px 0px 15px;
+		padding-right: 10px;
+		width: 235px;
+		line-height: 18px;
+	}
+	
+	.info-content .vision {
+		padding-left: 13px;
+		width: 235px;
+	}	
+	
+	.addressFooter .botty {
+		border-bottom: 1px solid black;
+		margin: 15px 21px 0px 21px;
+	}
+
+	.addy{
+		margin-top: 34px;
+		margin-left: 30px
+	}
+
+}
+/* So Ends the tablet CSS */
+
+
+/* So begins the mobile CSS */
+
+@media (max-width: 500px) {
+
+	.container {
+		max-width: 500px;
+	}
+
+	header {
+		height: 445px;
+		width: 500px;
+		position: relative;
+		background-image: none;
+		z-index: 10;
+		display: flex;
+	
+	}
+	
+	header nav {
+		margin-top: 70px;
+		height: 60px;
+		width: 100%;
+		display: flex;
+		position: relative;
+		line-height: 25px;
+		flex-direction: column;
+		align-items: center;
+		
+	}
+	
+	header nav a {
+		margin: 0px 0px 0 0px;
```

Comment:
when writing 0 in css you don't need to specify the unit of measurement, it automatically knows that 0 in every measurement is still 0. With that said, you could have just written this as margin 0 to save yourself some time.